commands/Transfering/transfer.js

```diff
@@ -36,8 +36,31 @@ function successMessage(res, amount){
   broadcastOperation(amount, user, transferred_to);
 }
 
+function canRemoveByAngryPoints(value) {
+  if (value > 0) {
+    return false
+  }
+  var angryPointsLimit = Libs.ResourcesLib.userRes("angryPointsLimit")
+  if (-value > angryPointsLimit.value()) {
```

Comment:
please add comment here like this:

// Master can remove points without exceeding the limit